tools/apiview/parsers/rust-api-parser/src/utils/lineIdUtils.ts

```diff
@@ -0,0 +1,50 @@
+import { PACKAGE_NAME } from "../main";
+import { ReviewLine } from "../models/apiview-models";
+
+export const lineIdMap = new Map<string, string>();
```

Comment:
Nit: IIRC, you're calling `item.id.toString()` everywhere where `id` is presumably numeric. Why not just have a `Map<number, string>`? Certainly cheaper.

Reply:
At some spots, I'm annotating them with suffixes like `_impl` to represent them. It helps to be consistent or else I'll have to maintain another property as such which would be worse.